### PR TITLE
[README] Modified

### PR DESCRIPTION
**Pull request by amino19**
**Description**
Modified `README` from given <a href="https://github.com/layer5io/layer5-repo-template">layer5-repo-template</a>.

This PR fixes #NA

**Notes for Reviewers**
Need review! 🧐

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+
+## What are Service Mesh Patterns?
+A service mesh is a layer in your infrastructure that facilitates communication between services… and so much more. Its value is enormous, and the value you derive from one is very much related to what role you play in the design, implementation, and operations of your cloud native applications and infrastructure.
+
+This feature area is under active development. Until additional documentation is published, see <a href="https://docs.google.com/document/d/1B2N78EdRiZF-yVo1-HY3syppwBBDumgMuYg6seD-AJ4/edit#"> the Meshery and Service Mesh Patterns</a> design specification for further details.
+
+## Prototyping Pattern Files and their Orchestration
+Interpreting Pattern files in Meshery
+<p align="center">
+    <img width="80%" src="https://lh3.googleusercontent.com/SNoWFfPQaqxHyxj_MlaqUkfox3K9tDTGSqLetSeGUZcSOVfQx8yKljmC1UwPQQBWcacPIqnNTgGq4S4TuKRfUns5c-sHwfZNoJZcu_Nrhc_WjvGFXS5BvjZZmWBexrnLkh_K1qEy">
```
Comment (leecalcote) [inline README.md:19]: @amino19 thanks for this PR. Will you add any images used into this folder - https://github.com/service-mesh-patterns/service-mesh-patterns/tree/master/.github/readme/images?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (amino19) [reply README.md:19]: @leecalcote : No sir, Even Layer5 <a href="https://layer5.io/company/brand">artwork</a> doesn't hold any image for service mesh patterns yet!
From where can I extract it out to put on the folder mentioned above? :confused:

**Comment by leecalcote** [comment]
@amino19 here you are, sir - https://drive.google.com/drive/u/0/folders/1X2NTTzJE3eh83Q985zMqoiIW71nq4GuX

**Comment by leecalcote** [comment]
@amino19, do you mind creating an issue to clean up some of the workflows that are not being used here?

**Pull request merged**